doc/en/api/1.0.0/opensearch-eo.yaml

```diff
@@ -15,6 +15,8 @@ paths:
   /collections:
     get:
       description: Retrieves a list of all available collections (names and links)
+      tags:
+       - "OpenSearchEo"
```

Comment:
Should be OpenSearchEO. Generally, when a bit of the name is an acronym already, it's kept in capitals.